src/pages/home.js

```diff
@@ -36,6 +37,7 @@ const Home = () => {
 console.log(getFirebase())
   return (
     <>
+      <MetaTags />
```

Comment:
All we need to do to get the meta tags on the homepage is to call the functi ... er' component.